src/main/java/com/palantir/gradle/versions/GradleWorkarounds.java

```diff
@@ -71,52 +71,50 @@ static boolean isConfiguring(ProjectState state) {
 
     /**
      * Allow a {@link ListProperty} to be used with {@link DomainObjectCollection#addAllLater}.
-     * <p>
-     * Pending fix: https://github.com/gradle/gradle/pull/10288
+     *
+     * <p>Pending fix: https://github.com/gradle/gradle/pull/10288
      */
     @SuppressWarnings("unchecked")
     static <T> ListProperty<T> fixListProperty(ListProperty<T> property) {
         Class<?> propertyInternalClass = org.gradle.api.internal.provider.CollectionPropertyInternal.class;
-        return (ListProperty<T>) Proxy.newProxyInstance(GradleWorkarounds.class.getClassLoader(),
-                new Class<?>[] {
-                        org.gradle.api.internal.provider.CollectionProviderInternal.class,
-                        ListProperty.class},
-                (proxy, method, args) -> {
-                    // Find matching method on CollectionPropertyInternal
-                    //org.gradle.api.internal.provider.CollectionProviderInternal
-                    if (method
-                            .getDeclaringClass() == org.gradle.api.internal.provider.CollectionProviderInternal.class) {
-                        if (method.getName().equals("getElementType")) {
-                            // Proxy to `propertyInternalClass` which we know DefaultListProperty implements.
-                            return propertyInternalClass.getMethod(method.getName(), method.getParameterTypes())
-                                    .invoke(property, args);
-                        } else if (method.getName().equals("size")) {
-                            return property.get().size();
-                        }
-                        throw new GradleException(String.format(
-                                "Could not proxy method '%s' to object %s",
-                                method,
-                                property));
-                    } else {
-                        return method.invoke(property, args);
-                    }
-                });
+        return (ListProperty<T>)
+                Proxy.newProxyInstance(
+                        GradleWorkarounds.class.getClassLoader(),
```

Comment:
Kinda weird to break after the cast?

```java
        return (ListProperty<T>) Proxy.newProxyInstance(
                    GradleWorkarounds.class.getClassLoader(),
```

Reply:
Fixed thanks to https://github.com/palantir/palantir-java-format/pull/11, see https://github.com/palantir/gradle-consistent-versions/pull/305/files#diff-56e0cf074cb9bc301934d0fd45ed3508R56-R59